packages/vue-client/src/views/GameView.vue

```diff
@@ -40,11 +41,15 @@ const variantDescriptionShort = computed(() => getDescription(variant.value));
 const user = useCurrentUser();
 const playing_as = ref<undefined | number>(undefined);
 const displayed_round = computed(() => view_round.value ?? current_round.value);
+const time_control = ref<ITimeControlBase | null>(null);
 
 function setNewState(stateResponse: GameStateResponse): void {
+  const { timeControl: timeControl, ...state } = stateResponse;
+  if (timeControl) time_control.value = timeControl ?? null;
```

Comment:
A little confused why we are checking `if (timeControl)` and also defining the `?? null` case.  Shouldn't one be sufficient?
```suggestion
  time_control.value = timeControl ?? null;
```

Reply:
You are right, I think I made this a little bit hastily. But I'm not sure if we want to assign null, so I'm thinking
`if (timeControl) time_control.value = timeControl;`